app/code/Magento/MessageQueue/etc/di.xml

```diff
@@ -43,6 +43,12 @@
             <argument name="lock" xsi:type="object">RefreshLock</argument>
         </arguments>
     </type>
+    <!-- @api -->
+    <virtualType name="shellBackground" type="Magento\Framework\Shell">
+        <arguments>
+            <argument name="commandRenderer" xsi:type="object">Magento\Framework\Shell\CommandRendererBackground</argument>
+        </arguments>
+    </virtualType>
```

Comment:
According to SVC failure, I guess we should move this part back to the CRON module, and maybe as a separate task move it to another. 
BTW, it looks like it should be in global di.xml which is located in app/etc, not in message queue module.

Reply:
Also there some functional test failures related to crons, are you sure you didn't broke anything?

Reply:
I've moved this to `app/etc/di.xml` as suggested. (It only appears to be used in `app/code/Magento/MessageQueue/etc/di.xml` as `Magento_Cron` now uses a different `virtualType`.)

I don't see which functional test failures you're referring to; can you share a link or explain what you're seeing? The functional test failures look out of scope for this pull request.

Reply:
From structure perpective it's the right place, but it causing SVC build failure, that has to be approved. Unfortunately, I never saw fast approvals on such failures :( 
My suggestion - move it back to Cron module for now, and maybe in a separate PR moving it to app/etc/di.xml.
What do you think?

Reply:
The "Semantic Version Checker" test failure is a false positive. The tool should be fixed. This pull request has already been open for more than two years.

From what I can tell, the failed functional test that you have linked is a race condition / poorly designed test. I will try to adjust the test now to ensure it works reliably.